test/upgrade.test.js

```diff
@@ -1,29 +1,26 @@
 'use strict'
 
-const { test, skip } = require('tap')
+const { test } = require('node:test')
 const Fastify = require('..')
 const { connect } = require('node:net')
 const { once } = require('node:events')
 const dns = require('node:dns').promises
 
 async function setup () {
   const localAddresses = await dns.lookup('localhost', { all: true })
-  if (localAddresses.length === 1) {
-    skip('requires both IPv4 and IPv6')
-    return
-  }
+  const doesNotSupportIPv6 = localAddresses.length === 1
```

Comment:
I think we can revert back to the original form and do:

```js
if (localAddresses.length === 1) {
	test.diagnostic('suite requires both IPv4 and IPv6')
	return
}
```

Reply:
```suggestion
  if (localAddresses.length === 1) {
    test.diagnostic('suite requires both IPv4 and IPv6')
    return
  }
```